### DISCUSSION TITLE
Will we be able to change the tab path at all?

### DISCUSSION THREAD

**Discussion by gerroon** [aleksey-hoffman/sigma-file-manager#301] [Q&A] [answered]
It looks like tabs are locked (or called locked tabs in other apps), any plans to provide a customizing option for that behavior?

**Comment by koppor** [comment]
This is also a real UI issue for me. From other apps (Web browsers such as Chrome or Firefox) or tab extensions for Windows Explorer (QTabBar), I am used that the path of a tab changes on update.

Since the tool does not update the path, the behavior feels a bit wired...

For Firefox, there is the concept of [Lock Tab Page](https://addons.mozilla.org/en-US/firefox/addon/lock-tab-page/). Maybe, this could also be used here?

**Reply by aleksey-hoffman** [reply, marked as answer]
Yeah, I see how it can feel quite unusual or even weird. The "changing tabs" will be the default behavior in v2
